src/ontology/mondo-edit.obo

```diff
@@ -559672,7 +559675,8 @@ synonym: "hallucinogen abuse with hallucinogen persisting perception disorder" N
 xref: ICD10CM:F16.183 {source="MONDO:relatedTo"}
 xref: MEDGEN:536590 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: UMLS:C0236720 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:536590"}
-is_a: MONDO:0024417 {source="https://orcid.org/0000-0001-5208-3432"} ! perceptual disorders
+is_a: MONDO:0001423 {source="PMID:29547576"} ! drug-induced mental disorder
+relationship: excluded_subClassOf MONDO:0024417 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-7638-4659"} ! perceptual disorders
```

Comment:
I am not 100% sure this is correct.
According to the definition, it is "A perceptual disorder caused by intoxication with hallucinogen drugs, especially LSD."
Is it both a perceptual disorder AND a drug-induced mental disorder? 
If if it not a perceptual disorder, the definition should be updated.